contributors/devel/sig-testing/monitoring.md

```diff
@@ -0,0 +1,264 @@
+# Monitoring Kubernetes Health
+
+**Table of Contents**
+
+- [Monitoring Kubernetes Health](#monitoring-kubernetes-health)
```

Comment:
nit: would it make sense to drop this so we can un-indent everything else?

Reply:
I don't get what you're saying here, can you please elaborate? 
Do you mean dropping the TOC altogether?

Reply:
I've updated the TOC. Pls review.